lib/livebook/teams.ex

```diff
@@ -223,34 +223,30 @@ defmodule Livebook.Teams do
   @doc """
   Encrypts the given value with Teams key derived keys.
   """
-  @spec encrypt(String.t() | nil, bitstring(), bitstring()) :: String.t()
-  def encrypt(value, _secret, _sign_secret) when value in ["", nil], do: value
+  @spec encrypt(String.t() | nil, bitstring()) :: String.t()
+  def encrypt(value, _secret) when value in ["", nil], do: value
 
-  def encrypt(value, secret, sign_secret) do
-    Plug.Crypto.MessageEncryptor.encrypt(value, secret, sign_secret)
+  def encrypt(value, secret) do
+    Plug.Crypto.MessageEncryptor.encrypt(value, secret, "unused")
   end
 
   @doc """
   Decrypts the given encrypted value with Teams key derived keys.
   """
-  @spec decrypt(String.t() | nil, bitstring(), bitstring()) :: {:ok, String.t()} | :error
-  def decrypt(value, _secret, _sign_secret) when value in ["", nil], do: value
+  @spec decrypt(String.t() | nil, bitstring()) :: {:ok, String.t()} | :error
+  def decrypt(value, _secret) when value in ["", nil], do: value
 
-  def decrypt(encrypted_value, secret, sign_secret) do
-    Plug.Crypto.MessageEncryptor.decrypt(encrypted_value, secret, sign_secret)
+  def decrypt(encrypted_value, secret) do
+    Plug.Crypto.MessageEncryptor.decrypt(encrypted_value, secret, "unused")
```

Comment:
@josevalim perhaps we could add `decrypt/2` and say that `decrypt/3` is for backward compatibility?

Reply:
Ah I see, yeah.